AlembicImporter/Packages/com.unity.formats.alembic/Documentation/com.unity.formats.alembic.md

```diff
@@ -1,55 +1,122 @@
+
+Alternatively, you can drag and drop the scene object with the ABC asset onto an empty area on Timeline area and create an Alembic Track with an Alembic Clip. If you are starting from an empty Timeline you will need to create the Alembic track first.
+
+![Timeline](images/timeline2.png)
+
+# Importing Alembic Files
+
+When you put ABC files in the Assets folder under your Unity Project, Unity automatically imports and stores them as Unity Assets. To view the import settings in the Inspector, click on the file in the Project window. You can customize how Unity imports the selected file by setting the properties on this window:
+
+![The Import Settings window](images/abc_import_options.png)
+
+| Property:| Function: |
+|:---|:---| 
+|__Normals__  <br/>"Read From File"<br/>"Compute If Missing"<br/> "Always Compute" <br/> "Ignore" |Defines if the normals from .abc file are used or calculate based on vertex position. The default "Compute If Missing" will use abc file normals, otherwise they will be calculated. |
+|__Tangents__ <br/> "Compute" <br/> "None"| Defines if the tangents are computed and is enabled by default. Since the ABC file has no tangent data, there are only 2 choices. However, please note that the calculation of tangents requires normals and UV data, and if these are missing the tangent cannot be computed.  Please note that the calculation of tangentsis expensive, so if not require then disabling of this option will increase the speed of playback.|
+|__Camera Aspect Ratio__ <br/> "Camera Aperture"<br/>"Default Resolution"<br/>"Current Resolution"| Defines whether to set the Unity Camera's aspect ratio. By default the ABC file will set the camera aspect ratio. The Alembic Importer uses the default resolution from the Player Settings. The Current Resolution refers to the aspect ratio define by the Screen. |
+|__Scale Factor__ | |
+|__Swap Handedness__ | |
+|__Interpolate Samples__ | |
+|__Swap Face Winding__ | |
+|__Turn Quad Edges__ | |
+|__Import Point Polygon__ | |
+|__Import Line Polygon__ | |
+|__Import Triangle Polygon__ | |
+|__Import Xform__ | |
+|__Import Camera__ | |
+|__Import Poly Mesh__ | |
+|__Import Points__ | |
+|__Time Range__ | |
+
+# Exporting Alembic Files
+
+The Alembic exporter supports exporting single frame and multi-frame Alembic files and can export game objects with the following components:
+* MeshRenderer
+* SkinnedMeshRenderer
+* ParticleSystem
+* Camera
+
+To configure a scene to export ABC file, add the AlembicExporter component to the appropriate game objects. The component can be configure to export the entire scene or individual objects.
+
+Using the AlembicExporter component automatically disable Draw Call Batching. If the Mesh group is valid after being batched then it the  will be exported, in some cases the data will be batched multiplied times and the results may change.  If you want to control the Batch settings they can be found in the Rendering section of Player Settings.

```

Comment:
disable**s**

Reply:
change 

>then it the  will be exported

to

>then it will be exported

Reply:
also I think there should be a period between "it will be exported" and "in some cases"

Reply:
what does this mean?

> data will be batched multiplied times